contrib/babelfishpg_common/src/encoding/mb/conv.c

```diff
@@ -225,17 +225,23 @@ TsqlUtfToLocal(const unsigned char *utf, int len,
 		unsigned char b2 = 0;
 		unsigned char b3 = 0;
 		unsigned char b4 = 0;
+		const unsigned char *cur;
 
-		/* "break" cases all represent errors */
+		/* Handle NULL character */
 		if (*utf == '\0')
-			break;
+		{
```

Comment:
These routines are being used to send the data of string data type with different collation to end client, how are we making sure that this change will not regress anything existing?